### ISSUE TITLE
Rename --atx-headings option to --atx-headers

### ISSUE THREAD

**Issue by adunning** [jgm/pandoc#5423]
It would be more precise to rename the `--atx-headers` option to `--atx-headings`. According to the Oxford American Dictionary, a [header](https://en.oxforddictionaries.com/definition/us/header) is a "a line or block of text appearing at the top of each page of a book or document" while a [heading](https://en.oxforddictionaries.com/definition/us/heading) is "a title at the head of a page or section of a book." (Presumably the current name could be kept as an alias?)

The term is also used somewhat inconsistently in the manual, which I am happy to sort out. (I apologize if there is another usage that I am missing here.)

**Comment by adunning** [comment]
Also affected: the option `--base-header-level`; and the extensions `blank_before_header`, `space_in_atx_header`, `header_attributes`, `implicit_header_references`, `mmd_header_identifiers`.

**Comment by jgm** [comment]
You are right, of course.  At least I got this right
in the commonmark spec.

But changing this in pandoc would be a major breaking
change, unless we left the old versions as deprecated
aliases.

In addition to this public-facing stuff, there are
internal things, e.g. the Header constructor in
pandoc's Block type.

**Comment by adunning** [comment]
No rush with it; I only just realized after adding the `block-headers` variable, and thought at first that I had the wrong name.

**Comment by adunning** [comment]
I think the most benefit comes from having correct terminology in the manual and option names. But if it would be useful to you, I would be happy to sort through the entire code base on some rainy day and correct the terms as necessary, to which you could then add aliases to prevent breaking old code.

**Comment by alerque** [comment]
I agree, this   should be changed and that the old flags should be left as an alias for legacy support.

In the mean time, isn't the title of this issue incorrect?
